### DISCUSSION TITLE
Need help understanding Soketi usage pricing in Cloudflare.

### DISCUSSION THREAD

**Discussion by tbell511** [soketi/pws#974] [Cloudflare Workers: Support]
I recently started testing your WebSocket solution in Cloudflare - so far it's been awesome! 🥳

However, I have some questions about the pricing, especially when considering scale. I am trying to determine the potential costs we might incur when using Soketi via Cloudflare at a larger scale, and compare it with our current provider, PieSocket.

Here's a breakdown of my current setup and the results from my initial tests:

**Sokeit Cloudflare Worker Setup:**

Mode: Unbound
Scalability: Horizontally Scalable
Cloudflare's documented pricing:

Duration: 400,000 GB-s included per month +$12.50 for every additional million GB-s
Requests: 1 million included per month +$0.15 for every additional million

**My Testing Results:**

Events Triggered: 25
Subscribed Clients: 2 (for about a 10 minute period)

**Cloudflare Worker UI showed:**
Requests: 1.99k
Duration: 102 GB-s

Considering the above, I'm surprised by the resource consumption for such a minimal test.

For context, we're currently using PieSocket, for which we pay $999 a month. This gives us:

90,000,000 messages a day (with the understanding that triggering 1 event to 10 subscribed clients would count as 10 messages, etc.)
40,000 concurrent connections

My main question is: Based on the Cloudflare pricing model and Soketi's architecture, how would the costs compare to our existing solution with PieSocket when operating at scale?

Any insights or clarifications would be greatly appreciated.

Thank you for your time, and once again, I've genuinely enjoyed exploring your product. Looking forward to your feedback.

**Comment by roopakv** [comment]
@tbell511 why did you close this discussion? Curious about this too
